src/v2/Apps/Partner/Components/__tests__/Overview/AboutPartner.jest.tsx

```diff
@@ -45,9 +56,30 @@ describe("AboutPartner", () => {
     expect(wrapper.find("Text").at(2).text()).toEqual(fullBio)
     expect(wrapper.find("Text").at(3).text()).toEqual(website)
     expect(wrapper.find("Text").at(4).text()).toEqual(`VAT ID#: ${vatNumber}`)
-    expect(wrapper.find("RouterLink").props().to).toEqual(
-      "http://www.theunitldn.com"
-    )
+    expect(wrapper.find("a").props().href).toEqual("http://www.theunitldn.com")
+  })
+
+  it("tracks correctly", () => {
+    const website = "http://www.theunitldn.com"
+    const slug = "the-unit-ldn"
+    const internalID = "1234asdf"
+
+    const wrapper = getWrapper({
+      Partner: () => ({
+        website,
+        slug,
+        internalID,
+        displayFullPartnerPage: true,
+      }),
+    })
+    wrapper.find("a").simulate("click")
+    expect(trackingSpy).toHaveBeenCalledWith({
+      action: "clickedPartnerLink",
+      context_owner_id: internalID,
+      context_owner_slug: slug,
+      context_owner_type: "partner",
+      destination_path: website,
+    })
```

Comment:
suggestion(non-blocking): what do you think of using toMatchInlineSnapshot here as we do in other analytics assertions? referring to [this comment](https://github.com/artsy/force/pull/9245#discussion_r792585840) from George

```javascript
expect(trackingSpy.mock.calls[0][0]).toMatchInlineSnapshot(`
  Object {
    "action": "clickedPartnerLink",
    "context_owner_id": "1234asdf",
    "context_owner_slug": "the-unit-ldn",
    "context_owner_type": "partner",
    "destination_path": "http://www.theunitldn.com",
  }
`)
```